admin_manual/ai/index.rst

```diff
@@ -69,3 +55,39 @@ It is based on points from these factors:
 If all of these points are met, we give a Green label. If none are met, it is Red. If 1 condition is met, it is Orange and if 2 conditions are met, Yellow.
 
 
+Features used by other apps
+---------------------------
+
+Some of our AI features are realized as generic APIs that any app can use and any app can provide an implementation for by registering a provider. So far, these are
+Machine translation, Speech-To-Text and Text processing.
+
+Machine translation
+^^^^^^^^^^^^^^^^^^^
+As you can see in the table above we have multiple apps offering machine translation capabilities. Each app brings its own set of supported languages.
+In downstream apps like the Text app, users can use the translation functionality regardless of which app implements it behind the scenes.
+
+Implementing apps
+~~~~~~~~~~~~~~~~~
+
+* *Text* for offering the translation menu
+* `Analytics <https://apps.nextcloud.com/apps/analytics>`_ for translating graph labels
+
+Speech-To-Text
+^^^^^^^^^^^^^^
+As you can see in the table above we have multiple apps offering Speech-To-Text capabilities. In downstream apps like the Talk app, users can use the transcription functionality regardless of which app implements it behind the scenes.
+
+Implementing apps
+~~~~~~~~~~~~~~~~~
+
+* `Speech-to-Text Helper <https://apps.nextcloud.com/apps/stt_helper>`_ for providing a Speech-To-Text smart picker
+* `Talk <https://apps.nextcloud.com/apps/spreed>`_ for transcribing calls (see `Nextcloud Talk docs <https://nextcloud-talk.readthedocs.io/en/latest/settings/#app-configuration>`_ for how to enable this)
+
+Text processing
+^^^^^^^^^^^^^^^
+As you can see in the table above we have multiple apps offering Text processing capabilities. In downstream apps like the Nextcloud Assistant app, users can use the text processing functionality regardless of which app implements it behind the scenes.
+
+Implementing apps
+~~~~~~~~~~~~~~~~~
+
+* `Assistant <https://apps.nextcloud.com/apps/assistant>`_ for various tasks
+* `Mail <https://apps.nextcloud.com/apps/mail>`_ for summarizing mail threads (can be enabled in the admin settings for the mail app)
```

Comment:
you could link to https://docs.nextcloud.com/server/latest/admin_manual/groupware/mail.html#thread-summary using an anchor :)